ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -88,6 +88,11 @@ Menu {
     delegate: StatusMenuItem {
         visible: root.hideDisabledItems ? enabled : true
         height: visible ? implicitHeight : 0
```

Comment:
btw, probably worth adding `width: ListView.view.width` here?

Reply:
> btw, probably worth adding `width: ListView.view.width` here?

Probably not bc it creates some kind of circular dependency. `ListView.view.width` depends on `Menu`'s `contentWidth`, which depends on `StatusMenuItem` implicit width (max along all instances). Even if it's not harmful it's not needed imo.